ui/app/AppLayouts/CommunitiesPortal/CommunitiesPortalLayout.qml

```diff
@@ -34,164 +35,247 @@ StatusSectionLayout {
         id: d
 
         property string searchText: ""
-        property int layoutVMargin: 70
-        property int layoutHMargin: 64
-        property int titlePixelSize: 28
-        property int subtitlePixelSize: 17
+
+        // values from the design
+        readonly property int layoutTopMargin: 10
+        readonly property int layoutBottomMargin: 249
+        readonly property int layoutHMargin: 64
+        readonly property int layoutWidth: 1037
+        readonly property int scrollViewTopMargin: 20
+
+        readonly property int titlePixelSize: 28
+        readonly property int subtitlePixelSize: 17
+
+        readonly property int preventShadowClipMargin: 40
 
         function navigateToCommunity(communityId) {
             root.communitiesStore.setActiveCommunity(communityId)
         }
+
+        readonly property bool searchMode: searcher.text.length > 0
+    }
+
+    // TODO: should be replaced with root.communitiesStore.curatedCommunitiesModel
+    // when backend is ready
+    CommunitiesPortalDummyModel {
```

Comment:
Could you move this in store in a curatedCommunitiesModel property and use it here through the store instead? (I'd expect also the .qml file to be inside the "stores" folder)